api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -272,7 +272,13 @@
     return ResponseEntity.ok(new EmptyResponse());
   }
 
-  private static String clusterNameForUser(DbUser user) {
+  /**
+   * Returns a name for the VM / cluster to be created for a given user in the workspace.
+   *
+   * @param user
+   * @return
+   */
+  public static String clusterNameForUser(DbUser user) {
```

Comment:
I'd externalize the prefix `all-of-us-`. I don't know from this statement what its scope is.

Also, one pet peeve I have is that if we're using hyphens to delineate subparts of an identifier, then it should be `allofus-`. Likely way too late.

Reply:
What do you mean by externalize? Bump to a constant in this file, out to a separate class, or all the way out to WorkbenchConfig?

I'm willing to find a separate class to home it in, to at least protect against circular dependencies.

Anything more probably isn't worth the effort. We haven't changed this value for the ~lifetime of the project, so I don't think it's a particularly sensitive constant.